prepare

```diff
@@ -75,19 +75,20 @@ if [ "$(curl -s -o /dev/null -w '%{http_code}' "$CUSTOM_ENV_CI_PROJECT_URL/-/raw
         # if a runner's basename starts with the key in a schutzfile,
         # it's assumed schutzfile[key] applies to that runner
         if [[ "$(basename "$CUSTOM_ENV_RUNNER")" =~ "$KEY".* ]]; then
-            REPOFILE=
-            REPOFILEPATH=$(echo "$SCHUTZFILE" | jq -r ."\"$KEY\".repos.file" -)
-            REPOS=$(echo "$SCHUTZFILE" | jq -r ."\"$KEY\".repos.$ARCH" -)
-            LENGTH=$(echo "$REPOS" | jq length -)
-            for i in $(seq 0 $((LENGTH - 1))); do
-                TITLE=$(echo "$REPOS" | jq -r .["$i"].title)
-                NAME=$(echo "$REPOS" | jq -r .["$i"].name)
-                BASEURL=$(echo "$REPOS" | jq -r .["$i"].baseurl)
-                REPOFILE+="[$TITLE]\nname=$NAME\nbaseurl=$BASEURL\nenabled=1\ngpgcheck=0\n"
+            REPOFILES_COUNT=$(echo "$SCHUTZFILE" | jq -r ."\"$KEY\".repos | length")
+            for j in $(seq 0 $((REPOFILES_COUNT - 1))); do
+                REPOFILEPATH=$(echo "$SCHUTZFILE" | jq -r ."\"$KEY\".repos[$j].file" -)
```

Comment:
This is weird, this line indexes `repos` as an array but the next line as an object.

Reply:
Ah yes, that's a mistake, thanks for pointing it out.